portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path

```diff
@@ -77,6 +77,11 @@
 	<td>//*[contains(@class, 'component-action btn btn-monospaced btn-unstyled')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DISPLAY_PAGE_TITLE</td>
+	<td>//*[contains(@data-qa-id, 'headerTitle')]</td>
```

Comment:
<td>//*[contains(@data-qa-id, 'headerTitle')]</td>
or 
	<td>//*[@data-qa-id='headerTitle']</td>
?

Reply:
Yes, you're right, the headerTitle id is unique, doesn't need to use contains. I'll modify it.